napari_skimage_regionprops/_parametric_images.py

```diff
@@ -1,7 +1,87 @@
+    """
+    import pandas as pd
+    import dask.array as da
+    from dask import delayed
+    from functools import partial
+
+    labels = labels_layer.data
+    table = pd.DataFrame(labels_layer.properties)
+
+    # special treatment for time series
+    if len(labels.shape) == 4:
+        # determine how the Frame column is called; in case there is any
+        frame_column = None
+        for potential_frame_column in ['frame', 'Frame']:
+            if potential_frame_column in table.keys():
+                frame_column = potential_frame_column
+                break
+
+        # Relabel one timepoint
+        output_sample = relabel_timepoint_with_map_array(labels, table, column, frame_column, 0)
+
+        lazy_arrays = []
+        for i in range(labels.shape[0]):
+            # build a delayed function call for each timepoint
+            lazy_processed_image = delayed(
+                partial(relabel_timepoint_with_map_array, labels, table, column, frame_column, i)
+            )
+            lazy_arrays.append(
+                lazy_processed_image()
+            )
+
+        # build an array of delayed arrays
+        dask_arrays = [
+            [da.from_delayed(
+                delayed_reader,
+                shape=output_sample.shape,
+                dtype=output_sample.dtype)]
+            if len(output_sample.shape) == 2
+            else da.from_delayed(
+                delayed_reader,
+                shape=output_sample.shape,
+                dtype=output_sample.dtype
+            )
+            for delayed_reader in lazy_arrays
+        ]
+        # Stack into one large dask.array
+        stack = da.stack(
+            dask_arrays,
+            axis=0)
+        return stack
+    else:
+        measurements = np.asarray(table[column]).tolist()
+        
+        # Ensure background measurements are present in measurements
+        # if also present in labels
+        if (0 not in table['label'].values) and (0 in labels):
+            measurements.insert(0, 0)
+        return relabel_skimage(labels, measurements)
+    
+def relabel_timepoint_with_map_array(labels, table, column, frame_column, timepoint):
+    labels_one_timepoint = labels[timepoint]
+    if frame_column is not None:
+        table_one_timepoint = table[table[frame_column] == timepoint]
+    else:
+        table_one_timepoint = table
+    measurements = np.asarray(table_one_timepoint[column]).tolist()
+    return relabel_skimage(labels_one_timepoint, measurements)
+
+def relabel_skimage(image, measurements):
+    from skimage.util import map_array
+    return map_array(image, np.unique(image), np.array(measurements))
+
+@deprecated("This function is deprecated! To adhere to future behavior and suppress this warning, use 'visualize_measurement_on_labels_with_map_array' instead")
 @register_function(menu="Measurement maps > Measurements on labels (nsr)")
 @register_function(menu="Visualization > Measurements on labels (nsr)")
```

Comment:
As we don't want people to continue using this function, and we have a replacement, we can remove it from the menu.

Reply:
OK, done!